junit-platform-console/src/main/java/org/junit/platform/console/tasks/Color.java

```diff
@@ -44,9 +44,8 @@ static Color valueOf(TestExecutionResult result) {
 				return Color.ABORTED;
 			case FAILED:
 				return Color.FAILED;
-			default:
-				return Color.NONE;
 		}
+		return Color.NONE;
```

Comment:
Having a switch statement for an enum that does not cover all enum constants is an error in my book. Consequently, the code should remain unchanged.

Reply:
@sbrannen Just to clarify, the reason I made this change is for the same reason I made the change for https://github.com/junit-team/junit5/pull/961/files#r134473571: error-prone requires that a switch statement that switches on the constants of an enum class, to not have a `default` case. This is so that error-prone can perform exhaustive enum-switch analysis via the built-in, turned-on-by-default rules ["UnnecessaryDefaultInEnumSwitch"](http://errorprone.info/bugpattern/UnnecessaryDefaultInEnumSwitch) and ["MissingCasesInEnumSwitch"](http://errorprone.info/bugpattern/MissingCasesInEnumSwitch).

Reply:
Although, having said that, it occurs to me that ["UnnecessaryDefaultInEnumSwitch"](http://errorprone.info/bugpattern/UnnecessaryDefaultInEnumSwitch) is only a warning-level rule rather than error-level, so it may make sense to disable the rule if it truly isn't working for us.